multimodal/src/autogluon/multimodal/data/preprocess_dataframe.py

```diff
@@ -120,7 +122,14 @@ def column_types(self):
 
     @property
     def image_path_names(self):
-        return self._image_path_names
+        if hasattr(self, "_image_path_names"):
+            return self._image_path_names
+        else:
+            return [col_name for col_name in self._image_feature_names if self._column_types[col_name] == IMAGE_PATH]
+
+    @property
+    def image_feature_names(self):
+        return self._image_feature_names if hasattr(self, "_image_feature_names") else self._image_path_names
```

Comment:
There is self._image_feature_names = [] in __init__. Seems it's not necessary to add the else statement.

Reply:
Thanks for pointing it out. Updated.